package.json

```diff
@@ -15,7 +19,8 @@
     "build": "react-scripts build",
     "test": "react-scripts test --watchAll=false --coverage --collectCoverageFrom=!**/serviceWorker.js",
     "coveralls": "cat ./coverage/lcov.info | node node_modules/.bin/coveralls",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "heroku-postbuild": "npm run build"
```

Comment:
@Nkoli `heroku-postbuild` runs after `npm run build`. What the postbuild script here does it to run the same `npm run build` again. This is not necessary, I think.

Also, Heroku suggests adding `yarn` to the engine option if you don't use npm to manage dependencies.